python/tk_framework_desktopserver/shotgun/scripts/cache_commands.py

```diff
@@ -85,26 +85,22 @@ def cache(cache_file, data, base_configuration, engine_name, config_data, config
         it is, then we include the __core_info and __upgrade_check commands.
     """
     engine = bootstrap(data, base_configuration, engine_name, config_data)
-
-    import sgtk
-    logger = sgtk.LogManager.get_logger(LOGGER_NAME)
-
-    logger.debug("Raw payload from client: %s", data)
+    engine.log_debug("Raw payload from client: %s" % data)
```

Comment:
can we add a comment here to help future code maintainers - just a note to say that the engine returned here may be linked to an older core etc.